demos/fab.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL FAB Demo</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <style>
+      /* Make the demo look a little bit better */
+      fieldset {
+        margin-bottom: 0.5rem;
+      }
+
+      fieldset .mdl-fab {
+        margin: 16px;
+      }
+    </style>
+    <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <main>
+      <h1>MDL FAB</h1>
+      <section>
+        <fieldset>
+          <legend>Normal FABs</legend>
+          <button class="mdl-fab">
+            <i class="material-icons mdl-fab__icon">favorite_border</i>
```

Comment:
Do we need this wrapper element if we just put the `material-icons` class on the button and then using

```scss
.mdl-fab {
  align-items: center;
  justify-content: center;
}
```

to position the text?

Also, perhaps we should put an `aria-label` attribute on the button element for accessibility purposes, since the fab only displays an icon.